scripts/artifactory-publish.sh

```diff
@@ -5,4 +5,18 @@ echo 'Deploying navi webservice models to artifactory'
 
 cd packages/webservice
 
-./gradlew -p models artifactoryPublish
+branch=`git branch --show-current`;
+
+echo $branch
```

Comment:
Maybe combine this with the last echo

```suggestion
echo "Deploying navi webservice models to artifactory for branch $branch"
```